terraform/application/config/review.yml

```diff
@@ -1,2 +1,3 @@
----
-EXAMPLE_KEY: example value 1
+RAILS_ENV: review
+RACK_ENV: review
+SECRET_KEY_BASE: abc123
```

Comment:
this should be a key vault secret not here! Please move to the app kv and change